### DISCUSSION TITLE
有没有考虑搞个railway版本的 感觉这里部署应该是最简单的

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [luolongfei/freenom#76] [Ideas]
![image](https://user-images.githubusercontent.com/38945550/166400688-4a712ee3-6e1e-4cc4-bae5-ea4f8e47c945.png)
这个是railway.app的后台管理界面 看日志 添加环境变量的设定都很方便

**Comment by luolongfei** [comment]
可以试试，然后提个 PR。

**Reply by DifferentSparkler** [reply]
> 可以试试，然后提个 PR。

而且部署到railway还有一个好处就是可以免费白嫖
![1651672194(1)](https://user-images.githubusercontent.com/38945550/166695399-a4864dea-d2c1-4d07-8351-ffaa69ebfc62.jpg)

有个问题 我不知道怎么提PR找了好久没有找到方法

**Reply by luolongfei** [reply]
这个，加上新功能后可用 [Pull requests](https://github.com/luolongfei/freenom/pulls)
